images/widgets/common/diagram/connector-end-arrow.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
Can we remove some garbage from svg, like this comment?

Reply:
done